fcos/release-checklist.md

```diff
@@ -74,6 +74,28 @@ At this point, Cincinnati will see the new release on its next refresh and creat
 
 - [ ] Wait for all releases that will be released simultaneously to reach this step in the process
 - [ ] Go to the [rollout workflow](https://github.com/coreos/fedora-coreos-streams/actions/workflows/rollout.yml), click "Run workflow", and fill out the form
+<details>
+<summary>Rollout general guidelines</summary>
+
+When setting a "Rollout start time" ask "when would be the best time to react to 
+any errors or regressions from updates?"
+Commonly we select 2PM UTC so that the rollout's start at 10am EST, but these can
+be fluid and adjust after talking with the fedora-coreos IRC.
+
+Additionally this is impacted by the day of the week or holidays, again ask 
+"when would be the best time to react to any errors or regressions from updates?"
+
+Commonly we try and release by Tuesday with 48h rollout timelines to insure
```

Comment:
Riskier rollouts might be 72h, and rapid rollouts (e.g. security fixes with few expected problems) might be 24h.

Reply:
Hmm, I added rows including them, but I am not sure I want to touch on why. I suspect risky and rapid release both would be talked about and the releasing engineer of fcos would be made aware of this.

Reply:
LMK, I think I like what I have now sorry for so much iteration.